cprover_bindings/src/goto_program/expr.rs

```diff
@@ -581,6 +585,28 @@ impl Expr {
         expr!(EmptyUnion, typ)
     }
 
+    /// `3.14f`
+    pub fn float16_constant(c: f16) -> Self {
+        expr!(HalfConstant(c), Type::float16())
+    }
+
+    /// `3.14159265358979323846264338327950288L`
+    pub fn float128_constant(c: f128) -> Self {
+        expr!(Float128Constant(c), Type::float128())
+    }
+
+    /// `union {_Float16 f; uint16_t bp} u = {.bp = 0x1234}; >>> u.f <<<`
```

Comment:
What is this comment saying?

Reply:
I was trying to replicate the comment for `float_constant_from_bitpattern` and `double_constant_from_bitpattern` , which seems to be a C union equivalent for those types.
Should I remove these comments, or can I add a clarifying comment saying that these are C union representations?